documentation/modules/ROOT/pages/installing-imagebaseinstall-cluster.adoc

```diff
@@ -158,7 +158,7 @@ Create the Installation ISO by running the following command:
 [.console-input]
 [source,bash,subs="attributes+,+macros"]
 -----
-~/openshift-install image-based create image --dir ibi-iso-workdir
+openshift-install image-based create image --dir ibi-iso-workdir
```

Comment:
Are we sure the binary is in the PATH?

Reply:
yes, I included in the same ansible task we are using to copy oc and kubectl to /usr/bin